repository-data/webfiles/src/main/resources/site/freemarker/common/location.ftl

```diff
@@ -0,0 +1,20 @@
+<#ftl output_format="HTML">
+<#include "../include/imports.ftl">
+
+<#-- @ftlvariable name="location" type="uk.nhs.digital.website.beans.Location" -->
+
+<#if location??>
+<div class="grid-wrapper grid-wrapper--article">
+    <div class="grid-row">
+        <div class="column column--reset">
+            <div class="article-header article-header--secondary">
+                <div class="grid-row">
+                    <div class="column column--two-thirds column--reset">
+                        <h1 data-uipath="person.firstname">${location.name}</h1>
```

Comment:
reference to person.firstname?  if UI for location doc type to come in separate ticket, then could remove this file altogether.